passiveData/build.gradle.kts

```diff
@@ -22,14 +23,23 @@ kotlin {
         }
     }
     sourceSets {
-        val commonMain by getting
+        val commonMain by getting {
+            dependencies {
+                implementation(Deps.KotlinX.dateTime)
+                api (Deps.KotlinX.serializationJson)
+            }
+        }
         val commonTest by getting {
             dependencies {
                 implementation(kotlin("test-common"))
                 implementation(kotlin("test-annotations-common"))
             }
         }
-        val androidMain by getting
+        val androidMain by getting {
+            dependencies {
+                api(Deps.AssessmentModel.sdk)
```

Comment:
If the dependencies end up being easier to riddle out by importing this into the commonMain, I'd say that makes sense. When (if ever) we are able to use a shared serialization for this it will be because Kotlin supports building frameworks with dependencies.

Reply:
Could you expand on "building frameworks with dependencies"? Does this mean KMM  can pull from Swift Package Manager and/or manage dependency tree in a Swift Package Manager compatible way?